src/nrnoc/hh.mod

```diff
@@ -94,7 +94,7 @@ PROCEDURE rates(v(mV)) {  :Computes rate and other constants at current v.
                       :Call once from HOC to initialize inf at resting v.
         LOCAL  alpha, beta, sum, q10
         : `TABLE minf` will be replaced with `:TABLE minf` if CoreNEURON enabled)
-        TABLE minf, mtau, hinf, htau, ninf, ntau DEPEND celsius FROM -100 TO 100 WITH 200
+        :TABLE minf, mtau, hinf, htau, ninf, ntau DEPEND celsius FROM -100 TO 100 WITH 200
```

Comment:
Probably forgot to avoid adding the changes to this file in the commit?

Reply:
Absolutely. Fixed!